data/places.json

```
@@ -125,5 +125,32 @@
     "coordinates": [36.955797, -121.972159],
     "date": "2022-07-04",
     "photo": "/photos/danielpereira/california.jpeg"
+  },
+  {
+    "author": "Gerson Júnior",
+    "type": "sticker",
+    "city": "Roma",
+    "country": "Italy",
+    "coordinates": [41.889048, 12.492502],
+    "date": "2022-07-11",
+    "photo": "/photos/gersonjunior/Coliseu.jpg"
+  },
+  {
+    "author": "Gerson Júnior",
+    "type": "sticker",
+    "city": "Napoli",
+    "country": "Italy",
+    "coordinates": [40.840007, 14.251875],
+    "date": "2022-07-08",
+    "photo": "/photos/gersonjunior/Napoles.jpg"
+  },
+  {
+    "author": "Gerson Júnior",
+    "type": "sticker",
+    "city": "San Giorgio, Albanese",
+    "country": "Italy",
+    "coordinates": [39.583670, 16.452920],
+    "date": "2022-07-04",
+    "photo": "/photos/gersonjunior/SanGiorgio.jpg"
```

```suggestion
    "photo": "/photos/GersosJ/SanGiorgio.jpg"
```

---

> ```suggestion
>     "photo": "/photos/GersosJ/SanGiorgio.jpg"
> ```
> 

Shouldn't it be "GresosJ"?